cmd/neofs-cli/modules/dump.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/mr-tron/base58"
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	controlSvc "github.com/nspcc-dev/neofs-node/pkg/services/control/server"
+	"github.com/nspcc-dev/neofs-sdk-go/util/signature"
+	"github.com/spf13/cobra"
+)
+
+const (
+	dumpFilepathFlag     = "path"
+	dumpIgnoreErrorsFlag = "no-errors"
+)
+
+var dumpShardCmd = &cobra.Command{
+	Use:   "dump",
+	Short: "Dump objects from shard",
+	Long:  "Dump objects from shard to a file",
+	Run:   dumpShard,
+}
+
+func dumpShard(cmd *cobra.Command, _ []string) {
+	key, err := getKey()
```

Comment:
i think we can use here (and in all `shard` section) `getKeyNoGenerate`

Reply:
Fixed.